.evergreen/buildvariants-and-tasks.in.yml

```diff
@@ -352,6 +352,7 @@ tasks:
           e2e_test_groups: <%= E2E_TEST_GROUPS.length %>
           e2e_test_group: <%= group.number %>
           debug: 'compass-e2e-tests*,electron*,hadron*,mongo*'
+          mongodb_version: latest-enterprise
```

Comment:
we didn't specify mongodb_version for multiple connections, so we got some default